zcncore/ethwallet.go

```diff
@@ -0,0 +1,301 @@
+package zcncore
```

Comment:
go test coverage 0.3% - could you pls add more tests?

Reply:
my latest commit is 67%